libs/shared/constants/src/lib/projectBasePath.ts

```diff
@@ -0,0 +1,7 @@
+export enum ProjectBasePath {
+  ServicePortal = '/minarsidur',
+  ConsultationPortal = '/samradsgatt',
+  ApplicationSystem = '/umsoknir',
+  AdminPortal = '/stjornbord',
+  CarRecycling = '/app/skilavottord',
+}
```

Comment:
Should these replace hardcoded paths for the projects?

Example:
![image](https://github.com/island-is/island.is/assets/6312838/7252b757-0295-4ff8-a587-040ba0cade46)
![image](https://github.com/island-is/island.is/assets/6312838/3b04bc61-69f2-4af6-a8c4-9e2488b20038)

Reply:
Honestly yes, these paths should be used where possible but since this is a hotfix PR I'll make another PR for main to do exactly that